svc/TrackService.js

```diff
@@ -10,8 +10,27 @@ import {stripTags} from 'hoist/utils/HtmlUtils';
 
 export class TrackService extends BaseService {
 
+    /**
+     * Create a Track Log entry.
+     * Client metadata is set automatically by the server's parsing of request headers.
+     *
+     * @param {Object|string} options - if a string, it will become the msg value.
+     * @param {string} [options.msg] - user-supplied message. Required if options is an object.
+     * @param {string} [options.message] - same as options.msg.  Just supporting msg & message. Use either, not both.
```

Comment:
Would avoid use of "user" here as that implies an end-user (ala client errors, where we do have a user-supplied message). Might suggest "Short description of the activity being tracked."

I would also vote for only including one of these (which is our favorite again? 😄) in the doc string. Could note in that comment "Can be passed as `msg` for backwards compatibility") Then don't surround that one in brackets. (It's technically optional in that you can pass the entire thing a string, but I think it makes sense to document it as a required prop of the object form....)

What happens if this doesn't end up getting passed a message? Does the server throw?